src/test/java/io/debezium/connector/yugabytedb/transforms/StriimCompatibleTest.java

```diff
@@ -0,0 +1,245 @@
+package io.debezium.connector.yugabytedb.transforms;
+
+import io.debezium.data.Envelope;
+import org.apache.kafka.connect.data.Schema;
+import org.apache.kafka.connect.data.SchemaBuilder;
+import org.apache.kafka.connect.data.Struct;
+import org.apache.kafka.connect.source.SourceRecord;
+import org.junit.jupiter.api.Test;
+
+import java.time.Instant;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.HashMap;
+import java.util.List;
+
+public class StriimCompatibleTest {
```

Comment:
Add a javadoc for this test as well. Take reference from other classes.

Reply:
Done.